src/patternfly/sass-utilities/scss-variables.scss

```diff
@@ -76,10 +76,6 @@ $pf-global--danger-color--300:          $pf-color-red-400 !default;
 $pf-global--primary-color--light-100:   $pf-color-blue-300 !default;
 $pf-global--primary-color--dark-100:    $pf-color-blue-400 !default;
 
-// Shadows
```

Comment:
Looks good, but I would add this `// Shadows` comment back, right above `//small` (on L79) so the shadow size comments have context of being under the "Shadows" section of this file.

Reply:
Good point! Thanks for taking a look at the code!